auth-web/src/composables/affiliations-factory.ts

```diff
@@ -82,6 +82,16 @@ export const useAffiliations = () => {
   /** Returns the status of the affiliation. */
   const status = (business: Business): string => {
     if (isTemporaryBusiness(business)) {
+      // Reflect 'Pending | Payment Incomplete' status
+      if (business.status === BusinessState.PENDING) {
```

Comment:
Would recommend moving this to the API if possible

Reply:
Arwen, did you do this because some combined states are more important than others and we want to show those?

Example 1: don't show future effective -- awaiting review is more important
Example 2: don't show future effective -- change requested is more important)?

If so, please make this clear in the code with sufficient comments.

Reply:
Yes, I will add comments

Reply:
Oh, sorry, I see you already have those comments further down (lines 92-93).

@seeker25 , what are you thinking should be in the API instead?

Reply:
return the enum that the UI needs and not create these pseudo statuses that live just in the UI, the result will be cleaner UI code that we could easily rewire up to a new UI if required

Reply:
There's already way too many conditional branches in the UI for this kind of thing @hfekete stated this before

Reply:
@seeker25 , please wait for the next commit. I believe a lot of this code will resolve to a single statement.

By the way, this code is not about creating pseudo-statuses -- it's about changing an enum to a user-friendly string. Do  you agree that this belongs in the UI?

PS - If I'm not mistaken, this code is handling the response of the `affiliations` call.

Reply:
Why not just return the correct status enum from the API, then map that to a UI user friendly string?

Reply:
> Why not just return the correct status enum from the API, then map that to a UI user friendly string? No conditions needed

I believe that is exactly what the code is doing presently. `BusinessState` is a string lookup enum:

![image](https://github.com/user-attachments/assets/59da0c65-7cd4-4ad9-8b15-b7ac7387c8d5)

Reply:
Looks conditional to me and not just looking at the enum:

![image](https://github.com/user-attachments/assets/e1455840-acf4-474d-86c1-59a89f25f8a3)

Reply:
This logic is being rewritten. Please wait for the next commit.

Reply:
anyways just my thoughts, I've seen people try to refactor this code and it proved very difficult